omop/migrations/0002_cdm_core.py

```diff
@@ -0,0 +1,310 @@
+# Generated migration: create OMOP CDM core tables for Eunomia GiBleed (v5.3)
+from django.db import migrations
+
+SQL_UP = r"""
```

Comment:
hmmm, any idea WHY the vanilla SQL is used here?

did you use some options to force it?

And is it really needed here?

I thing the Django ORM-like instructions is more safe / more flexible